server/src/relay/RelayHttpServer.go

```diff
@@ -201,7 +204,7 @@ func parseCommandLine() (relayParams librelay.RelayParams) {
 	relayParams.DefaultGasPrice = *defaultGasPrice
 	relayParams.GasPricePercent = big.NewInt(*gasPricePercent)
 	relayParams.RegistrationBlockRate = *RegistrationBlockRate
-	relayParams.EthereumNodeURL = *ethereumNodeUrl
+	relayParams.ethereumNodeURL = *ethereumNodeURL
```

Comment:
Not sure this refactoring should be in this PR, or even how it compiles as relayParams has no such member.